apps/sv/frontend/src/components/form-components/ConfigField.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Divider, TextField as MuiTextField, Typography } from '@mui/material';
+import { useFieldContext } from '../../hooks/formContext';
+import { ConfigChange } from '../../utils/types';
+
+export interface ConfigFieldProps {
+  configChange: ConfigChange;
+}
+
+export const ConfigField: React.FC<ConfigFieldProps> = props => {
```

Comment:
I've abstracted each config field into this new form-component so it can be used by dso and amulet forms